test/test_custom_ops.py

```diff
@@ -1803,9 +1785,11 @@ def test_delayed_error_no_requires_grad(self):
 
 optests.generate_opcheck_tests(
     MiniOpTest,
-    ["aten", "MiniOpTest"],
-    mini_op_test_failures_dict,
-    "test/test_custom_ops.py",
+    ["aten", "mini_op_test"],
+    get_file_path_2(
```

Comment:
what's this helper function 🤔

Reply:
Something about internal fbcode having different codepaths to the resource :/